packages/curve-common/src/features/switch-chain/ui/SettingsButton.tsx

```diff
@@ -0,0 +1,18 @@
+import Button from '@mui/material/Button'
+import SettingsIcon from '@mui/icons-material/Settings'
+import Typography from '@mui/material/Typography'
+import { t } from '@lingui/macro'
+
+export const SettingsButton = (props: { onClick: () => void }) => (
+  <Button color="ghost" size="small" onClick={props.onClick}>
+    <SettingsIcon sx={{ fontSize: 22, fill: 'transparent', stroke: 'currentColor' }} />
+    <Typography
+      sx={{ marginLeft: 1, verticalAlign: 'top' }}
```

Comment:
verticalAlign doesn't do anything, and the marginLeft of 1 is pretty arbitrary. Personally I'd increase it even a bit more.

Also, why not use the `startIcon` property of the MUI Button component?

Reply:
`marginLeft: 1` is the same as `marginLeft: t => t.spacing(1)` which is actually 
I'll see if the `startIcon` prop works better